pkg/builder/webhook_test.go

```diff
@@ -735,6 +811,12 @@ func (*TestCustomDefaulter) Default(ctx context.Context, obj runtime.Object) err
 
 var _ admission.CustomDefaulter = &TestCustomDefaulter{}
 
+func mutatorFactoryForTestDefaulter(scheme *runtime.Scheme) admission.HandlerFactory {
+	return func(obj runtime.Object, _ admission.Decoder) admission.Handler {
+		return admission.WithCustomDefaulter(scheme, obj, &TestCustomDefaulter{}).Handler
+	}
+}
```

Comment:
This is very confusing. We're using a mutator, but returning a custom defaulter handler?

Reply:
I moved this somewhere else, but I'm still using a CustomDefaulter. Let me know if you prefer adding a different handler.

Reply:
I added a custom handler.